src/main/java/com/geronimo/model/Message.java

```diff
@@ -21,6 +21,7 @@
     public Message(String text, User author) {
         this.text = text;
         this.author = author;
+        author.addMessage(this);
```

Comment:
Let's not modify 3rd party entity from a constructor of another entity, let's move this logic to the service, e.g. in postMessage we can add a message to an author if we really need to before saving.